plugins/module_utils/data_set.py

```diff
@@ -1007,17 +1013,22 @@ def create(
         """
         original_args = locals()
         formatted_args = DataSet._build_zoau_args(**original_args)
-        response = datasets._create(**formatted_args)
-        if response.rc > 0:
+        try:
+            datasets.create(**formatted_args)
```

Comment:
Just a note: The same issue does NOT arise in line 182 in the `ensure_present` method because it calls the internal `DataSet.create` function defined by the  class in this file (`module_util/datasets.py`) and not the ZOAU python API (`datasets.create`) which changed in  v1.3.0.

Reply:
hey Ketan, this seems to be the best approach since our ansible interface keeps using the word "type" instead of "dataset_type". Thanks.